src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+import { timeout } from '../../utils';
+import { Formik, Form } from 'formik';
+import { FieldGroup, InputFieldItem } from '../form-helpers/form-helpers';
+
+const fieldNames = {
+  email: 'email'
+};
+
+/** Prepare empty values for all fields
+ * It is required because in another way, the fields are not marked as touched.
+ */
+const getFormInitialValues = () =>
+  Object.keys(fieldNames).reduce(
+    (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
+    {},
+  );
+
+const ForgotPassword = ({ intl }) => {
+
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const validate = (values) => {
+    const errors = {};
+
+    if (!values[fieldNames.email]) {
+      errors[fieldNames.email] = _('validation_messages.error_required_field');
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values[fieldNames.email])) {
+      // TODO: review if this is our desired validation
+      errors[fieldNames.email] = _('validation_messages.error_invalid_email_address');
+    }
+
+    return errors;
+  };
+
+  const onSubmit = async (values, { setSubmitting }) => {
+    // TODO: implement login submit
+    await timeout(1500);
+    setSubmitting(false);
+  };
+
+  return (
+    <main className="panel-wrapper">
+      <article className="main-panel">
+        <header>
+          <h1 className="logo-doppler-new">Doppler</h1>
+          <div className="language-selector">
```

Comment:
Maybe in another PR, but, what do you think about moving this code into a component? (We already meet [the rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)))

Reply:
Yes i think it's better